jumpscale/clients/stellar/stellar.py

```diff
@@ -529,8 +539,11 @@ def _transfer(
             )
 
         horizon_server = self._get_horizon_server()
+        if fund_transaction and asset_code == "TFT":
```

Comment:
not just TFT, more assets are supported

Reply:
what assets are supported? So that I check on them, because if the asset code check was omitted, and fund transaction was true the base fee will be set to 0. eg: when transfer of  XLMs

Reply:
In this part only I removed the check so that if there is funding the base fee will always be set to 0 no matter what the asset it , and later on the asset itself is checked